content/en/account_management/billing/custom_metrics.md

```diff
@@ -7,64 +7,206 @@ aliases:
+
+| Metric Name       | Tag Values             | Aggregations                            |
+| ----------------- | ---------------------- | --------------------------------------- |
+| `request.Latency` | `host:A`, `endpoint:X` | `count`, `sum`, `min`, `max`, and `avg` |
+| `request.Latency` | `host:A`, `endpoint:Y` | `count`, `sum`, `min`, `max`, and `avg` |
+
+[1]: /developers/metrics/types/?tab=histogram
+{{% /tab %}}
+{{% tab "Distribution" %}}
+
+**A DISTRIBUTION metric generates five custom metrics** for each unique combination of metric name, host, and tag values to support the server-side aggregations  `count`, `sum`, `min`, `max`, and `avg`. [Learn more about DISTRIBUTION metric type][1].
+
+Suppose you have a distribution metric measuring `request.Latency` in which the following tag value combinations are present in your data as shown below:
+
+{{< img src="account_management/billing/custom_metrics/request_latency.png" alt="Request latency" responsive="true" style="width:80%;">}}
+
+The number of unique combinations of metric name, host, and tag values submitted for a `GAUGE` metric with this tagging scheme is 4:
+
+| Metric Name       | Tag Values                           |
+| ----------------- | ------------------------------------ |
+| `request.Latency` | `host:A`, `endpoint:X`, `status:200` |
+| `request.Latency` | `host:B`, `endpoint:X`, `status:200` |
+| `request.Latency` | `host:B`, `endpoint:X`, `status:400` |
+| `request.Latency` | `host:B`, `endpoint:Y`, `status:200` |
+
+Given `request.Latency` is submitted as a `DISTRIBUTION`, Datadog [stores 5 custom metrics][1] for each of the original 4 unique tag value combinations to account for server side-aggregations: `count`, `sum`, `min`, `max`, and `avg`. This results in a total of 20 custom metrics for this `DISTRIBUTION` metric:
+
+| Metric Name       | Tag Values                           | Aggregations                            |
+| ----------------- | ------------------------------------ | --------------------------------------- |
+| `request.Latency` | `host:A`, `endpoint:X`, `status:200` | `count`, `sum`, `min`, `max`, and `avg` |
+| `request.Latency` | `host:B`, `endpoint:X`, `status:200` | `count`, `sum`, `min`, `max`, and `avg` |
+| `request.Latency` | `host:B`, `endpoint:X`, `status:400` | `count`, `sum`, `min`, `max`, and `avg` |
+| `request.Latency` | `host:B`, `endpoint:Y`, `status:200` | `count`, `sum`, `min`, `max`, and `avg` |
+
+##### Distribution Metrics with Additional Percentile Aggregations
+
+[Adding additional percentile aggregations][2]: `p50`, `p75`, `p90`, `p95`, `p99`) to your DISTRIBUTION metric increases the number of distinct custom metrics by raising the ammount of aggregations. The final number of metric is:
+
+```
+Number_of_distinct_tag_combinations * ( 5 + Number_of_additional_percentile_aggregations_choosen)
```

Comment:
This formula looks funky -- I got push back for displaying an exact formula anyway so let's stick with giving examples.